units/systemd-boot-update.service

```diff
@@ -0,0 +1,24 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Automatic Boot Loader Update
+Documentation=man:bootctl(1)
+DefaultDependencies=no
+Conflicts=shutdown.target
+After=local-fs.target
+Before=sysinit.target shutdown.target systemd-update-done.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart={{BINDIR}}/bootctl --no-variables --graceful update
```

Comment:
```suggestion
ExecStart={{ROOTBINDIR}}/bootctl --no-variables --graceful update
```

I guess?

Reply:
{{ROOTBINDIR}} ain't gonna work either, this is not an .in file ;P

Please just just drop the path fully.

Reply:
relying on PATH for executing binaries we invoke is still a really bad idea, I still think we shouldn't have changed our code to use that.

Reply:
> relying on PATH for executing binaries we invoke is still a really bad idea

I think it makes sense to rely on PATH for *external* tools, but for our internal binaries using the full path is preferrable.